app/forms/waste_carriers_engine/declare_convictions_form.rb

```diff
@@ -4,9 +4,11 @@ module WasteCarriersEngine
   class DeclareConvictionsForm < BaseForm
     attr_accessor :declared_convictions
 
+    validates :declared_convictions, "waste_carriers_engine/yes_no": true
+
     def initialize(transient_registration)
       super
-      self.declared_convictions = @transient_registration.declared_convictions
+      self.declared_convictions = transient_registration.declared_convictions
```

Comment:
NIT Adding a line between super and this as per other changes

```suggestion
      
      self.declared_convictions = transient_registration.declared_convictions
```